tensorflow/python/data/experimental/kernel_tests/ignore_errors_test.py

```diff
@@ -166,6 +167,18 @@ def testCardinality(self):
     ds = dataset_ops.Dataset.range(10).apply(error_ops.ignore_errors())
     self.assertEqual(self.evaluate(ds.cardinality()), dataset_ops.UNKNOWN)
 
+class IgnoreErrorsCheckpointTest(checkpoint_test_base.CheckpointTestBase,
+                                 parameterized.TestCase):
+
+  def _build_ds(self):
+    return dataset_ops.Dataset.range(5).map(
+        array_ops.ones).map(lambda x: array_ops.gather(x, [0])).apply(
+            error_ops.ignore_errors())
+
+  @combinations.generate(test_base.default_test_combinations())
+  def testIgnoreErrorsCore(self):
+    num_outputs = 4
+    self.run_core_tests(self._build_ds, num_outputs)
```

Comment:
now that we use multiple inheritance, it would make sense to rename `run_core_tests` to `run_checkpoint_tests` (in a separate PR)

Reply:
sure will take it up once all checkpoint tests have been moved.